packages/ui/src/forum/components/PostList/PostListItem.tsx

```diff
@@ -23,7 +27,26 @@ interface PostProps {
 
 export const PostListItem = forwardRef<HTMLDivElement, PostProps>(({ post, isSelected, isPreview }, ref) => {
   const { createdAtBlock, updatedAt, author, text, reaction, repliesTo } = post
-  const edited = useMemo(() => updatedAt && <EditionTime>(edited {relativeTime(updatedAt)})</EditionTime>, [updatedAt])
+  const { showModal } = useModal()
+  const time = useMemo(() => {
+    if (!updatedAt) {
+      return null
+    }
+
+    return (
+      <EditionTime
+        onClick={() =>
+          showModal<PostHistoryModalCall>({ modal: 'PostHistory', data: { postId: post.id, author: author } })
+        }
+      >
+        (edited{' '}
+        {differenceInHours(new Date(), new Date(updatedAt)) >= 24
+          ? formatDateString(updatedAt)
+          : relativeTime(updatedAt)}
```

Comment:
This could go to a (local or shared) helper method. Some name suggestions: `formattedPastDate`, `relativeIfRecent`.